deluge/plugins/AutoAdd/deluge/plugins/autoadd/gtkui.py

```diff
@@ -216,58 +217,58 @@ def on_get_enabled_plugins(result):
         else:
             iter = self.accounts.append()
             self.accounts.set_value(iter, 0, client.get_auth_user())
-            self.glade.get_widget('OwnerCombobox').set_active(0)
-            self.glade.get_widget('OwnerCombobox').set_sensitive(False)
+            self.main_builder.get_object('OwnerCombobox').set_active(0)
+            self.main_builder.get_object('OwnerCombobox').set_sensitive(False)
 
     def set_sensitive(self):
         maintoggles = ['download_location', 'append_extension',
                        'move_completed', 'label', 'max_download_speed',
                        'max_upload_speed', 'max_connections',
                        'max_upload_slots', 'add_paused', 'auto_managed',
                        'stop_at_ratio', 'queue_to_top', 'copy_torrent']
-        [self.on_toggle_toggled(self.glade.get_widget(x + "_toggle")) for x in maintoggles]
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expression "[self.on_toggle_toggled(self.main_builder.get_object((x) + ('_toggle'))) for x in maintoggles]" is assigned to nothing](https://www.codacy.com/app/tbkizle/deluge/file/2419162533/issues/source?bid=3238201&fileBranchId=3238205#l229)